CONTRIBUTING.md

```diff
@@ -17,7 +17,24 @@ limitations under the License.
 
 # Contributors Guide
 
- If you believe that you have found a bug, please search for an existing [issue](https://issues.apache.org/jira/browse/accumulo) to see if it has already been reported. If you would like to add a new feature to Accumulo, please send an email with your idea to the [dev](mailto:dev@accumulo.apache.org) mail list. If it's appropriate, then we will create a ticket and assign it to you.
+ If you believe that you have found a bug, please search for an existing [issue](https://github.com/apache/accumulo/issues) to see if it has already been reported. If you would like to add a new feature to Accumulo, please send an email with your idea to the [dev](mailto:dev@accumulo.apache.org) mail list. If it's appropriate, then we will create a ticket and assign it to you.
+
+## Issues
+
+For pull request and issues, the following labels are used. Consider an issue
+with labels `v1.9.3`, `v2.0.2`, and `bug`.  If the issue is open, then its a
+bug that someone plans to fix in 1.9.3 and 2.0.2.  If it is closed, then
+it was fixed in those versions.
+
+ Label       | Description
+-------------|----------------------------------------------------------------------------
+ blocker     | Indicates a release blocker that must be fixed in labeled versions.
+ bug         |
+ duplicate   |
+ enhancement |
+ vX.Y.Z      | Fix version. The prefix `v` was chosen so that these labels sort last in the drop down list.
```

Comment:
Using github milestones was considered, however an issue can only have one milestone.  This is incompatible with Accumulo's workflow.  The advantage of using milestones over labels is they can be closed after a release.  Closing the milestone makes it unavailable for use and more importantly hides it in the drop down list. We will not be able to close the labels and will end with a large number over time.  This is why a prefix was chosen that makes the labels sort to the end of the drop down list.  The sorting makes browsing non-version labels easy.  Github's search in the label drop down makes finding a specific version quick.

Since this is the Accumulo project I tried using `~` as a prefix for version labels, however github ignored this when sorting 😞